### PR TITLE
fix: use owner as CREATE2 salt

### PR DESCRIPTION
**Pull request by PaulRBerg**
Addresses https://github.com/cantinasec/review-sablier2/issues/25.

### CODE REVIEW

File: src/PRBProxyRegistry.sol [hunk 1]
```diff
@@ -247,35 +227,18 @@ contract PRBProxyRegistry is IPRBProxyRegistry {
...
-        // forgefmt: disable-next-line
-        emit DeployProxy({
-            origin: tx.origin,
-            operator: msg.sender,
-            owner: owner,
-            seed: seed,
-            salt: salt,
-            proxy: proxy
-        });
+        emit DeployProxy({ origin: tx.origin, operator: msg.sender, owner: owner, proxy: proxy });
```
Comment (andreivladbrg) [inline src/PRBProxyRegistry.sol:242]: Is it still neede to emit origin: tx.origin here?

File: src/PRBProxyRegistry.sol [hunk 2]
```diff
@@ -139,37 +136,20 @@ contract PRBProxyRegistry is IPRBProxyRegistry {
...
```
[the remark this answers is not in the document]
Comment (PaulRBerg) [reply src/PRBProxyRegistry.sol:152]: Good catch!!

File: src/PRBProxyRegistry.sol (hunk 1 above)
Comment (PaulRBerg) [reply src/PRBProxyRegistry.sol:242]: Nope.

https://github.com/PaulRBerg/prb-proxy/pull/123#discussion_r1238342252

**Review by tibbarytsur — APPROVED** [review]

**Comment by PaulRBerg** [comment]
Yanked in favor of https://github.com/PaulRBerg/prb-proxy/pull/130.

**Pull request merged**